### PR TITLE
Swap Ubuntu 12.04 testing for openSUSE Leap

### PR DESCRIPTION
**Pull request by tas50**
Swap Ubuntu 12.04 testing for openSUSE Leap

    Ubuntu 12.04 is going end of life so lets swap testing for another OS we should check.

    While we’re in here lets update a few other things
    -  There’s no longer a reason to hold back dokken to the 1.0 release.
    - Remove a dupe travis config
    - Remove centos 5 from the kitchen config since it’s going into the mostly EOL phase
    - Update platforms in the local kitchen config

    Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -193,7 +174,6 @@ matrix:
       - cd kitchen-tests
     script:
       - bundle exec kitchen test webapp-debian-8
-    bundler_args: --without ci development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
```
Comment (lamont-granquist) [inline .travis.yml]: that line of red looks wrong?

File: kitchen-tests/.kitchen.travis.yml [hunk 2]
```diff
@@ -43,16 +42,6 @@ platforms:
       - RUN /usr/bin/apt-get update
       - RUN /usr/bin/apt-get -y install zlib1g-dev sudo net-tools wget ca-certificates
 
-- name: centos-5
```
Comment (thommay) [inline kitchen-tests/.kitchen.travis.yml]: definitely we should not remove that.

File: kitchen-tests/.kitchen.travis.yml [hunk 3]
```diff
@@ -2,7 +2,6 @@
 driver:
   name: dokken
   privileged: true
-  chef_image: chef/chef
```
Comment (lamont-granquist) [inline kitchen-tests/.kitchen.travis.yml]: what was this there for?  beware that i had to do some hacky shit to make dokken work with chef_github provisioner, and applying standardized dokken config to this file may cause issues...

File: kitchen-tests/.kitchen.travis.yml (hunk 2 above)
Comment (tas50) [inline kitchen-tests/.kitchen.travis.yml]: We weren't actually using this in the config. It was just defined in the Kitchen yaml.

File: kitchen-tests/.kitchen.travis.yml (hunk 3 above)
Comment (tas50) [inline kitchen-tests/.kitchen.travis.yml]: Old kitchen-dokken uses sean's image. The last pile of releases have defaulted to our images so this config becomes redundant.

**Comment by tas50** [comment]
I'll open a PR for just the opensuse changes later. Most of this was just merged anyways in a diff PR

**Pull request closed without merging**